### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by Sigevio**

### CODE REVIEW

**Comment by Sigevio** [comment]
image

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Build CI
+
+on: [pr, push]
```
Comment (pavelskipenes) [inline .github/workflows/build.yml:3]:
```suggestion
on: [pull_request, push]
```

**Comment by pavelskipenes** [comment]
Here is the documentation https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions

**Review by pavelskipenes — APPROVED** [review]
build succeeded

**Pull request merged**